### ISSUE TITLE
t.el.parentNode is null

### ISSUE THREAD

**Issue by CyberMike97** [Dogfalo/materialize#5809]
I get the following error: "t.el.parentNode is null on line
materialize.min.js:6:74966", randomly when i click on a button and when i open or close a modal. After i get the error i'm unable to interact with the page and i have to reload it.

**Comment by Dogfalo** [comment]
Do you have an example of this on a Codepen we can look into?

**Comment by CyberMike97** [comment]
I have used the examples on the docs. After doing some tests I noticed that the error occurs when I execute these two instructions:
`M.Toast.dismissAll();`
`M.toast({html: "mytext",classes:"rounded"});`

On Chrome 65 the error is the following:
```
Uncaught TypeError: Cannot read property 'removeChild' of null
    at Object.complete (materialize.min.js:6)
    at o (materialize.min.js:6)
    at r (materialize.min.js:6)
    at Object.D.v.tick (materialize.min.js:6)
    at e (materialize.min.js:6)
```

**Comment by Dogfalo** [comment]
Fixed in ff6f9fbd8ecd399a409f2ab9e59c4b1b303fd91c
